serverless-test-project/package.json

```diff
@@ -19,5 +19,9 @@
     "test": "ALARM_TOPIC=test sls package",
     "test:v2": "ALARM_TOPIC=test sls package -c serverless-v2.yml"
   },
-  "gitHead": "c4300240f9e854c4eb1c5503b839882ff0cd1cae"
+  "gitHead": "c4300240f9e854c4eb1c5503b839882ff0cd1cae",
+  "dependencies": {
+    "https": "^1.0.0",
```

Comment:
Node.js has a built-in `https` module, so I don't think we need to add an NPM module. Does it add anything?
I'd like to suggest replacing this with `axios` in any case and changing the handler to use `async`/`await`.